src/current/v24.2/performance-recipes.md

```diff
@@ -297,6 +303,16 @@ A low percentage of live data can cause statements to scan more data ([MVCC valu
 
 Reduce the [`gc.ttlseconds`]({% link {{ page.version.version }}/configure-replication-zones.md %}#gc-ttlseconds) zone configuration of the table as much as possible.
 
+### KV DistSender batches being throttled (performance impact to larger clusters)
+
+If you see `distsender.batches.async.throttled` values that aren't zero (or aren't consistently near zero), experiment with increasing the [KV layer DistSender]({% link {{ page.version.version }}/architecture/distribution-layer.md %}#distsender) and [KV layer Streamer]({% link {{ page.version.version }}/architecture/distribution-layer.md %}#streamer) concurrency using the `kv.streamer.concurrency_limit` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}).  In 24.3, these default values were increased by 6x and 12x, respectively.  For versions older than 24.3, increasing the value by 6x and 12x would be a good starting point.
+
+To validate a successful result, you can increase this value until you see no new throttled requests AND no increase in tail latency (e.g. `p99.999`).
+
+This does increase the amount of RAM consumption per node to handle the increased concurrency, but it's proportional to the load and an individual flow's memory consumption should not be significant. Bad outcomes include increased tail latency or too much memory consumption with no decrease in the number of throttled requests. 
+
+Changing this setting can increase risk of [out of memory (OOM) errors]({% link {{ page.version.version }}/cluster-setup-troubleshooting.md %}#out-of-memory-oom-crash) depending on the value of [`cockroach start --max-go-memory`]({% link {{ page.version.version }}/cockroach-start.md %}#flags-max-go-memory) and/or [`GOMEMLIMIT`](https://pkg.go.dev/runtime#hdr-Environment_Variables).
```

Comment:
I think this last line is overstating the risk.  On a system with 32 or 64GB of RAM, adding a max of ... :sticks finger up in the air: ~10MB of RAM will be unnoticed given the increase in throughput.  And, given we've bumped the default, we shouldn't be alarmist, either.  Arguably we should be offsetting the amount of RAM from connections queued behind this limit with the increase in consumption so it should be a wash.  Certainly in my testing, I saw no increase in RAM utilization that was significant or observable.

Reply:
ok - dropped it altogether

> I'm worried we're making a mountain out of a molehill with this change and the drag-along concerns. 😆

fair enough :-D 

also moved this to just the v24.3 'Performance Recipes' page per your other comment